test/integration/answer/AnswerIT.java

```diff
@@ -1,332 +0,0 @@
-/*
```

Comment:
why is this test gone now? We don't have any tests for Answer now right? If we don't want it get we should @ignore it and then add it to the issue you're making with all the tests to re-enable

Reply:
All of the integration tests were out of date and would be unbuildable, not just untestable. They can be migrated in at a later time right? They still exist in master.

Reply:
ok good thanks for making issue for these